plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2069,6 +2083,84 @@ public void testSplitPruningForFilterOnPartitionColumn()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testAllAvailableTypes()
+    {
+        assertUpdate("CREATE TABLE test_all_types (" +
+                " bool BOOLEAN" +
```

Comment:
nit: can you keep column naming consistent and not cryptic. E.g for name column for type `<TYPE>`: `<type>_column`

Reply:
or `a[n]_<type>`

Reply:
as in https://github.com/trinodb/trino/blob/3611b0cdd16b330f31d5dd824bc8c03a2a9893fe/testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveSparkCompatibility.java#L53-L68

Reply:
ok, sorry for that